Dockerfile

```diff
@@ -66,6 +66,7 @@ RUN wget https://github.com/GoogleCloudPlatform/terraformer/releases/download/0.
 # kics-scan ignore-line
 COPY --from=build_env /app/bin/kics /app/bin/kics
 COPY --from=build_env /app/assets/queries /app/bin/assets/queries
+COPY --from=build_env /app/assets/cwe_csv/* /app/bin/assets/cwe_csv
```

Comment:
kindly perform this same update into this [dockerfiles](https://github.com/Checkmarx/kics/tree/master/docker) if needed